docker-compose.yml

```diff
@@ -143,6 +158,8 @@ services:
       - PORT=${PAYMENT_SERVICE_PORT}
       - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
       - OTEL_RESOURCE_ATTRIBUTES=service.name=paymentservice
+    depends_on:
```

Comment:
I think it should

Reply:
@Frapschen, I think this same PR should be used to fix the frontend dependencies (as suggested above). Otherwise, the frontend service can start before for example checkoutservice and probably won't work before checkoutservice is ready for traffic.

Reply:
I have added, pls check again

Reply:
> the frontend service can start before for example checkoutservice and probably won't work before checkoutservice is ready for traffic.

and this might be a cool scenario to demonstrate how distributed tracing can be used to understand what has happened.

Reply:
Yes, I can see them now. My mistake.